web/telemetry/publisher.go

```diff
@@ -40,7 +40,7 @@ func (tp *TelemetryPublisher) Publish(telemetryName string, installationID uuid.
 	return nil
 }
 
-var apiHost = "https://telemetry.trento-project.io" // this service still does not exist
+var apiHost = "https://telemetry.trento.suse.com" // this service still does not exist
```

Comment:
We can remove the comment I guess, that service is being spawned up as I understood.

Reply:
Fair 'nuff